app/_layouts/extension.html

```diff
@@ -103,20 +130,51 @@
 {% endcapture %}
 
 {% capture plugin_config_for_service %}
-### Enabling the plugin on a Service
+### Enabling the plugin on a service
 
-{% if page.params.yaml_examples == false and page.params.k8s_examples == false %}
+{% if page.params.yaml_examples == false and page.params.k8s_examples == false and page.params.manager_examples == false and page.params.konnect_examples == false %}
 
-{{ plugin_config_for_service_admin_api }}
+  {{ plugin_config_for_service_admin_api }}
 
 {% else %}
 
 {% navtabs %}
 {% navtab Admin API %}
 
-{{ plugin_config_for_service_admin_api }}
+  {{ plugin_config_for_service_admin_api }}
+
+  `<service>` is the `id` or `name` of the service that this plugin
+    configuration will target.
+
+{% endnavtab %}
+{% unless page.params.manager_examples == false %}
+{% navtab Kong Manager %}
+Configure this plugin on a service:
+
+1. In Kong Manager, select the workspace.
+2. From the Dashboard, scroll down to **Services** and click **View** for the service row.
+3. Scroll down to plugins and click **Add Plugin**.
+4. Find and select the **{{ page.name }}** plugin.
+
+    <div class="alert alert-ee blue">
+    <strong>Note:</strong> If the plugin is greyed out, then it is not available
+    for your product tier. See <a href="">Kong Gateway tiers</a>.
```

Comment:
```suggestion
    for your product tier. See <a href="/enterprise/latest/deployment/licensing/">Kong Gateway tiers</a>.
```
Applies to the rest as well.